walletlibrary/src/main/java/com/microsoft/walletlibrary/did/sdk/datasource/network/apis/HttpAgentOpenId4VciApi.kt

```diff
@@ -18,13 +18,14 @@ internal class HttpAgentOpenId4VciApi(
 ) {
     companion object {
         const val OPENID4VCI_INTER_OP_PROFILE = "oid4vci-interop-profile-version=0.0.1"
+        private val openId4VciInteropHeadersMap = mapOf(Constants.PREFER to OPENID4VCI_INTER_OP_PROFILE)
```

Comment:
It is better to keep the value alone in a constant, especially in this case, since we will have more than 1 prefer headers. We will be concatenating them to a comma separated value before adding to the map. Having the value separately will make the processing simpler.